.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+name: ci
+on: [push, pull_request]
+env:
+  COVERAGE_OPTION: ./node_modules/.bin/nyc
+  NODE_VERSION: 10
+  PARSE_SERVER_TEST_TIMEOUT: 20000
+jobs:
+  check-mongo:
+    strategy:
```

Comment:
to avoid infinite CI, add also a ` timeout-minutes: 60`

Reply:
done